### PR TITLE
Replicate lufdetan feinstaub v1 routes

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

Fixes missing code/functionality for these routes.

- [x] `/v1/sensor/{sensor_id}/ `  - all measurements of the last 5 minutes for one sensor
- [x] `/v1/filter?city=&country=&type=` - 5 minutes filtered by query
- [x] `/static/v2/data.json` - average of all measurements per sensor of the last 5 minutes for all sensors
- [x] `/static/v2/data.1h.json` - average of all measurements per sensor of the last hour
- [x] `/static/v2/data.24h.json` - average of all measurements per sensor of the 24 hours
- [x] `static/v2/data.dust.min.json` - same as data.json but dust sensors only
- [x] `/static/v2/data.temp.min.json` - same as data.json but temp

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: sensorsafrica/api/v1/router.py [hunk 1]
```diff
@@ -9,6 +9,8 @@
     SensorDataView,
 )
```
Comment (DavidLemayian) [inline sensorsafrica/api/v1/router.py]: Interesting choice of file name.. I would expect this to be called `urls.py` following Django's naming style. Did they change this?

Comment (karimkawambwa) [reply sensorsafrica/api/v1/router.py]: > I'm probably talking jibberish but would we also include this in our urls.py? - https://github.com/opendata-stuttgart/feinstaub-api/blob/master/feinstaub/sensors/urls.py

This is just a method for us to verison routes. 
There exists a urls.py that uses these routers.

File: sensorsafrica/management/commands/cache_static_json_data.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            '24h': ['../../../static/v2/data.24h.json']
+        }
+        cursor = connection.cursor()
+        cursor.execute('''
+            SELECT sd.sensor_id, sdv.value_type, AVG(CAST(sdv."value" AS FLOAT)) as "value", COUNT("value"), sd.location_id
+                FROM sensors_sensordata sd
+                    INNER JOIN sensors_sensordatavalue sdv
+                        ON  sd.id = sdv.sensordata_id WHERE "timestamp" >= (NOW() - interval %s)
+                GROUP BY sd.sensor_id, sdv.value_type, sd.location_id
+        ''', [intervals[options['interval']]])
```
Comment (kilemensi) [inline sensorsafrica/management/commands/cache_static_json_data.py]: Pulling `24 hours` worth of data from DB _might_ bring the system to its knees... Can't we _append_ the hourly data to a 24 hour data file?

Comment (karimkawambwa) [reply sensorsafrica/management/commands/cache_static_json_data.py]: After testing this, locally point to production. 
For the moment, I dont' see any issues.

**Pull request merged**